test/test_ljson_mem.py

```diff
@@ -88,14 +88,16 @@ def test_unique_check():
 	with pytest.raises(ValueError):
 		table.additem({"age": 12, "name": "chris",
 				"lname": "griffin"})
+	table.rows.pop()
```

Comment:
I would instead use something like:

```suggestion
data_ = copy.copy(data)
table = Table(..., data_)
```